packages/@o3r/eslint-plugin/src/rules/o3r-widget-tags/o3r-widget-tags.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { TSESLint } from '@typescript-eslint/experimental-utils';
+import o3rWidgetTagsRule, { O3rWidgetTagsRuleOption } from './o3r-widget-tags';
+
+const ruleTester = new TSESLint.RuleTester({
+  parser: require.resolve('@typescript-eslint/parser'),
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module'
+  }
+});
+
+const code = `
+export interface Config extends Configuration {
+  /**
+   * @o3rWidget supportedType
+   * @o3rWidgetParam supportedParam "valid value"
+   */
+  prop: string;
+}
+`;
+
+const options: O3rWidgetTagsRuleOption[] = [{
+  widgets: {
```

Comment:
You don't update the documentation? Isn't that the options for the rule defined in the rules.md?